packages/craftcms-webpack/index.js

```diff
@@ -0,0 +1,347 @@
+// Fix issue with monorepo and some plugins
+// https://github.com/jantimon/html-webpack-plugin/issues/1451#issuecomment-712581727
+const _require = (id) => require(require.resolve(id, { paths: [require.main.path] }));
+
+const path = require("path");
+const glob = require("glob");
+const { merge } = require("webpack-merge");
+const dotenv = require("dotenv");
+const fs = require("fs");
+const yargs = require("yargs/yargs");
+const { hideBin } = require("yargs/helpers");
+const argv = yargs(hideBin(process.argv)).argv;
+const Dotenv = require("dotenv-webpack");
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+const { CleanWebpackPlugin } = require("clean-webpack-plugin");
+const VueLoaderPlugin = require('vue-loader/lib/plugin');
+const { WebpackManifestPlugin } = _require('webpack-manifest-plugin');
+
+// Where webpack-cli was run from
+const rootPath = path.resolve("./");
+
+/**
+ * Returns the first existing file based on a list of paths.
+ *
+ * @param {string[]} paths
+ * @returns {string|undefined}
+ */
+const getFirstExistingPath = (paths = []) => {
+  return paths.find((path) => {
+    return fs.existsSync(path);
+  });
+};
+
+/**
+ * Returns and array of webpack configs.
+ *
+ * @param {string} globPattern
+ * @param {Object} options
+ * @param {string} options.cwd Path where the glob should be executed from.
+ */
+const getConfigs = (
+  globPattern = "src/web/assets/*/webpack.config.js",
+  options = {
+    cwd: module.parent.path,
```

Comment:
@timkelty Should we rename this to be like the other changes you made?

```suggestion
    context: module.parent.path,
```

Reply:
This is not required because we are just passing the glob options straight through